calculus/source/02-DF/01.ptx

```diff
@@ -112,17 +112,17 @@
     </activity>
     
  <definition xml:id = "def-inst-vel"> 
-     <p> The instanteous velocity at <m>t=a</m> is the limit as <m>h \to 0</m> of the difference quotient <m>\frac{f(a+h)-f(a)}{h}</m>. In the activity above the instantaneous velocity at <m>t=2</m> is given by the limit
+     <p> The instanteous velocity at <m>t=a</m> is the limit as <m>h \to 0</m> of the difference quotient <m>\dfrac{f(a+h)-f(a)}{h}</m>. In the activity above the instantaneous velocity at <m>t=2</m> is given by the limit
```

Comment:
```suggestion
     <p> The <term>instantaneous velocity</term> at <m>t=a</m> is the limit as <m>h \to 0</m> of the difference quotient <m>\dfrac{f(a+h)-f(a)}{h}</m>. In the activity above the instantaneous velocity at <m>t=2</m> is given by the limit
```

Reply:
You didn't do this, but I'd prefer the "activity above" comment to be separated out into a separate observation/remark.

Another enhancement while we're here: use $\lim_{h\to 0}$ notation in the definition, not just words.